crimsobot/utils/cringo.py

```diff
@@ -191,9 +226,23 @@ def __init__(self) -> None:
     return '<:blank:589560784485613570>\n' + '\n'.join(final_string)
 
 
+# this emoji marks the card
+async def marker(cardsize: int, row: int, column: int) -> str:
```

Comment:
don’t make it a coroutine if it doesn’t need to be. just a regular function will do, no async